### PR TITLE
Added tooltip for editable fields in the "System Settings"

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Added tooltip for editable fields in the "System Settings". Thanks to @GulomovCreative and @sergant210.

**What it looks like:**
tooltip_1

In MODX tables, you can change the value by double-clicking, but this is not obvious to the user.

_Unfortunately, this PR adds this functionality ONLY to the "System Settings", we need to extend this behavior for all editable fields in MODX tables. But I do not have enough knowledge, need help._

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/14176

### CODE REVIEW

File: core/lexicon/en/default.inc.php [hunk 1]
```diff
@@ -130,6 +130,7 @@
 $_lang['edit_snippet'] = 'Edit Snippet';
 $_lang['edit_template'] = 'Edit Template';
 $_lang['edit_tv'] = 'Edit TV';
+$_lang['editable_field'] = 'Double-click for quick editing.';
```
Comment (JoshuaLuckers) [inline core/lexicon/en/default.inc.php]:
```suggestion
$_lang['editable_field'] = 'Double-click to open edit mode.';
```

I believe this is a more accurate description for the "double-click" action. Sometimes you are able to edit the value directly, sometimes it shows a select box you have to click if you want to change the value.

**Review by Mark-H — APPROVED** [review]
Please make sure to mark PRs as ready for review when they're done - I'm going ahead and approving it, but it wont be able to get merged until you mark it as ready.

**Comment by Ruslan-Aleev** [comment]
@Mark-H There is also a question. Can this behavior be somehow extended for all grids via modx.grid.js?

**Comment by Mark-H** [comment]
@Ruslan-Aleev I'm not entirely sure - if so it would likely need to go into `_loadColumnModel()` as that prepares renderers and editors, but I din't immediately see how/where you'd need to add the class to it there.

**Comment by Ruslan-Aleev** [comment]
@Mark-H Clear. We can insert the same code separately in each script with a grid, but this is inconvenient :)

**Comment by Mark-H** [comment]
I've played around with this and almost have a new PR ready that consistently applies it to any grid 🎉 

I'm thinking the bold styling is a little too much though. Maybe you @Ruslan-Aleev or someone else has a better idea on how to style it so it stands out, but isn't in your face _too much_? Maybe applying the link color on hover, or only doing the tooltip?

**Comment by Ruslan-Aleev** [comment]
Good news, thanks! We will look forward to PR :)

As for styles, in addition to tooltips, I think they are still needed to identify property in line (although not so much needed,).
We can, as you rightly pointed out, change the color, or, another option, reduce the transparency (opacity: .7;)
Additional styles will be useful for mobile devices, it seems to me.

**Comment by Mark-H** [comment]
PR created: #14748

**Pull request closed without merging**